app/EventBus/EventBusServiceProvider.php

```diff
@@ -88,6 +89,7 @@ function () use (&$eventBus, $container): void {
                             OwnershipSearchProjector::class,
                             OwnershipPermissionProjector::class,
                             ImageLDProjector::class,
+                            SendMailsForOwnership::class,
```

Comment:
I think there is one big drawback with this approach. The event listeners are also processed synchronously. This means that the call to create, approve or reject an ownership is blocked until the mail is set.

Maybe we need to look into the approach of an export where an asynchronous command is used with a dedicated worker. Check for the usage of `createResqueCommandBus`

Reply:
Maybe we need an extra step, basically a conversion between the event and an async command to send a mail. So the `OwnershipRequested` is handled by service that create an async `SendMail` command.

Reply:
All of this has been implemented, with the necessary headache :-)